README.md

```diff
@@ -298,6 +298,16 @@ cargo llvm-cov --no-report --features b
 cargo llvm-cov --no-run --lcov
 ```
 
+In combination with the `show-env` subcommand, coverage can also be produced from arbitrary binaries:
+
+```sh
+cargo llvm-cov clean --workspace # remove artifacts that may affect the coverage results
+source <(cargo llvm-cov show-env --export-prefix)
+cargo build # build rust binaries
+# commands using binaries in target/debug/*, including `cargo test`
+cargo llvm-cov --no-run --lcov
```

Comment:
```suggestion
# ...
cargo llvm-cov --no-run --lcov # generate report without tests
```

(Clarify which line the comment "commands using binaries..." is associated with.)